src/components/Project/ImageAnalysis/ImageViewer.vue

```diff
@@ -179,33 +188,15 @@ function handleEdit(event) {
   // Re-attach the edit handling logic
   newLine.on('pm:edit', handleEdit)
 
-  findPixelCoords(event.layer.getLatLngs())
+  requestLineProfile(event.layer.getLatLngs())
 }
 
-// Refactored code to calculate line length in pixels
-function findPixelCoords(latLngs) {
+function requestLineProfile(latLngs) {
```

Comment:
how do `startCoordinates` and `endCoordinates` know what the start and end are if the user has zoomed in on the image? does `getLatLngs` take care of it?

Reply:
Yeah! Did some testing to confirm but it seems like leaflet doesn't care about what zoom level you're at and will read the same lat lngs when you click on the same point on the image

Reply:
oh no way! that's awesome